### PR TITLE
fix: Log store operations as system generated logs

### LINKED ISSUE #24882: [Bug]: store value execution shows up as user log instead of system log
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Store value function is logged as a user-generated one instead of system generated as is the case with showAlert, closeModal etc...

Applicabe to `storeValue`, `removeValue` and `clearStore`

This behaviour causes clutter when using a filter to observe console statements.

### Steps To Reproduce

1. trigger a storeValue function and observe console and system logs

image

### Public Sample App

_No response_

### Environment

Production

### Issue video log

_No response_

### Version

Cloud

### PR DESCRIPTION
**Pull request by arunvjn**
## Description
- Changed store operation logs from user logs to system generated logs.
- Replaced left icon button in debugger with ADS Icon component.
- Prevents the log from expanding when the click is on the link.
- Makes the entire log clickable when there is additional collapsed information
>
> Links to Notion, Figma or any other documents that might be relevant to the PR
>
>
#### PR fixes following issue(s)
Fixes #24882 

#### Type of change
- Bug fix (non-breaking change which fixes an issue)
>
>
#### How Has This Been Tested?
- [x] Manual
- [x] Cypress
>
>
#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/cypress/support/Pages/DebuggerHelper.ts [hunk 1]
```diff
@@ -156,6 +159,11 @@ export class DebuggerHelper {
     this.agHelper.GetNAssertContains(this.locators._errorCount, count);
   }
 
+  changeLogsGroup(option: string) {
```
Comment (Aishwarya-U-R) [inline app/client/cypress/support/Pages/DebuggerHelper.ts:162]: nit: TS Helper methods are PascalCase. `ChangeLogsGroup`

**Review by Aishwarya-U-R — COMMENTED** [review]
Approving Cypress changes.

**Review by Rishabh-Rathod — DISMISSED** [review]
Changes LGTM.

**Review by ChandanBalajiBP — DISMISSED** [review]
LGTM

**Pull request merged**